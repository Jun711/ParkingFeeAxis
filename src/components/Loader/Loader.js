import React from 'react'
import { View, ActivityIndicator } from 'react-native'
import styles from './LoaderStyles'
import { LOADER_COLOR } from "../../util/constants"

export const Loader = () => {
  return (
    <View style={styles.container}>
      <ActivityIndicator style={styles.loadingIndicator} size='large' color={LOADER_COLOR}/>
    </View>
  )
}

export default Loader
