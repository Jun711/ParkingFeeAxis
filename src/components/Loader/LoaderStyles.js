import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'flex-start',
    paddingTop: 10
  }
})

export default styles