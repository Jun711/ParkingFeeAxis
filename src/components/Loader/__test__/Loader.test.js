import 'react-native';
import React from 'react';
import Loader from '../Loader';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('<Loader />', () => {
  it('FaqList renders correctly', () => {
    const LoaderTree = renderer.create(
      <Loader />
    ).toJSON();
    expect(LoaderTree).toMatchSnapshot();
  });
});

