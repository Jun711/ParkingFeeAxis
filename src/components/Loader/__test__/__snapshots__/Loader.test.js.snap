// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> FaqList renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
      "paddingTop": 10,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#515c6d"
    hidesWhenStopped={true}
    size="large"
    style={undefined}
  />
</View>
`;
